cmake/projects/Boost/atomic/hunter.cmake

```diff
@@ -18,5 +18,5 @@ hunter_download(
     Boost
     PACKAGE_COMPONENT
     atomic
-    PACKAGE_INTERNAL_DEPS_ID "31"
+    PACKAGE_INTERNAL_DEPS_ID "32"
```

Comment:
Update for `PACKAGE_INTERNAL_DEPS_ID` in `hunter.cmake.in` is missing.

Reply:
ups... sorry ... done.